app/views/admin/link_check_reports/show.js.erb

```diff
@@ -0,0 +1,3 @@
+<% if @report.completed? %>
```

Comment:
If the report hasn't been completed yet should we be showing a progress indicator? Or perhaps this does not deserve to be in this file?

Reply:
@thomasleese from my understanding the way the JS works/will work is if the poll to the show endpoint returns any content it executes the returned js. So, in this case, it will only replace the loading spinner with links/success if the report is complete, which is desired.